packages/swapkit/cli/src/util/useSwapKit.ts

```diff
@@ -0,0 +1,23 @@
+import type { SwapKitCore as SKC } from '@swapkit/core';
+import { keystoreWallet } from '@swapkit/wallet-keystore';
+import { createContext, useState } from 'react';
+
+import type { Config } from './useConfig.js';
+const { SwapKitCore } = await import('@swapkit/core');
+
+export const SwapKitContext = createContext<{
+  swapkit: SKC;
+  keystoreConnected: boolean;
+  setKeystoreConnected: (connected: boolean) => void;
+}>({ swapkit: new SwapKitCore(), keystoreConnected: false, setKeystoreConnected: () => {} });

```

Comment:
thinking of usage react there... Maybe we could use some easier part for global storage 🤔 like zustand + immer only